diff --git a/Makefile b/Makefile
index 11eedbb..fdfa277 100644
--- a/Makefile
+++ b/Makefile
@@ -5,10 +5,8 @@ INSTALL = install
 INSTALL_PROGRAM = $(INSTALL)
 INSTALL_DATA = $(INSTALL) -m 644
 
-prefix = /usr/local
-exec_prefix = $(prefix)
-bindir = $(exec_prefix)/bin
-datarootdir = $(prefix)/share
+bindir = $(PREFIX)/bin
+datarootdir = $(PREFIX)/share
 mandir = $(datarootdir)/man
 man1dir = $(mandir)/man1
 
